server/embed/config_test.go

```diff
@@ -92,136 +92,99 @@ func TestConfigFileOtherFields(t *testing.T) {
 	assert.Equal(t, false, cfg.SocketOpts.ReuseAddress, "ReuseAddress does not match")
 }
 
-func TestConfigFileFeatureGates(t *testing.T) {
-	testCases := []struct {
-		name                                string
-		serverFeatureGatesJSON              string
-		experimentalStopGRPCServiceOnDefrag string
-		experimentalInitialCorruptCheck     string
-		expectErr                           bool
-		expectedFeatures                    map[featuregate.Feature]bool
-	}{
-		{
-			name: "default",
-			expectedFeatures: map[featuregate.Feature]bool{
-				features.DistributedTracing:      false,
-				features.StopGRPCServiceOnDefrag: false,
-				features.InitialCorruptCheck:     false,
-			},
-		},
+func defaultFeatureGates() map[featuregate.Feature]bool {
```

Comment:
Can you move util functions below the main test? Going from top to bottom in hierarchy is more readable.

Reply:
Thanks, updated